release_build_files/readme.md

```diff
@@ -45,14 +45,32 @@ structure.
 Directories               | Contents
 ------------------------- | --------
 include                   | C++ headers
-frameworks/ios/ARCH       | iOS frameworks (compiled against libc++)<br>A multi-architecture framework is<br>provided in the *universal* directory.
-libs/ios/ARCH             | iOS static libraries (compiled against<br>libc++)<br>Multi-architecture libraries are<br/>provided in the *universal* directory.
-libs/android/ARCH/STL     | Android (GCC 4.8+ compatible) static<br>libraries for each architecture and STL variant.<br>_STL variants available:_<br><ul><li>`c++`: LLVM libc++ runtime (recommended)</li><li>`gnustl`: GNU STL (deprecated)</li></ul><br>More information can be found in the<br>[NDK C++ Helper Runtimes](https://developer.android.com/ndk/guides/cpp-support.html#runtimes) documentation.
+frameworks/ios/ARCH       | iOS frameworks (compiled against libc++)
```

Comment:
Are we still using iOS frameworks @jonsimantov ? 

I believe we enabled xcframework. And directories should be `xcframeworks/api/ARCH`
The daily packaging artifact also denotes that we are using xcframework.
https://github.com/firebase/firebase-cpp-sdk/actions/runs/831150353

Reply:
No, you're right, ios is xcframeworks only.
darwin still uses frameworks.

Reply:
Updated to xcframeworks.